test/integration/overload_integration_test.cc

```diff
@@ -0,0 +1,94 @@
+#include "test/integration/http_protocol_integration.h"
+
+#include "absl/strings/str_cat.h"
+#include "absl/strings/substitute.h"
+
+namespace Envoy {
+
+class OverloadIntegrationTest : public HttpProtocolIntegrationTest {
+protected:
+  OverloadIntegrationTest()
+      : injected_resource_filename_(TestEnvironment::temporaryPath("injected_resource")),
+        file_updater_(injected_resource_filename_) {}
+
+  void initialize() override {
+    config_helper_.addConfigModifier([this](envoy::config::bootstrap::v2::Bootstrap& bootstrap) {
+      const std::string overload_config = absl::Substitute(R"EOF(
+        refresh_interval {
+          seconds: 0
+          nanos: 1000000
+        }
+        resource_monitors {
+          name: "envoy.resource_monitors.injected_resource"
+          config {
+            fields {
+              key: "filename"
+              value {
+                string_value: "$0"
+              }
+            }
+          }
+        }
+        actions {
+          name: "envoy.overload_actions.stop_accepting_requests"
+          triggers {
+            name: "envoy.resource_monitors.injected_resource"
+            threshold {
+              value: 0.9
+            }
+          }
+        }
+      )EOF",
+                                                           injected_resource_filename_);
+      const bool success = Protobuf::TextFormat::ParseFromString(
+          overload_config, bootstrap.mutable_overload_manager());
+      ASSERT(success);
```

Comment:
Probably want to use ASSERT_TRUE (from gtest) so `success` gets checked during release build tests.

Reply:
Thanks - done.